style.css

```diff
@@ -442,3 +462,393 @@ ul > li > i {
   margin-top: 22px;
   text-align: center;
 }
+
+@media only screen and (min-width: 992px) {
+  body {
+    font-family: 'Poppins', sans-serif;
+    margin: 0;
+    background-color: #e5e5e5;
+  }
+
+  header {
+    width: 100%;
+    height: 916px;
+    background-color: #fff;
+  }
+
+  .top-navigation {
+    width: 1440px;
+    height: 72px;
+  }
+
+  .logo {
+    width: 100px;
+    height: 40px;
+    margin: 25px 0 0 155px;
+  }
+
+  .item-navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    text-decoration: none;
+    width: 259px;
+    height: 40px;
+    margin-right: 142px;
+  }
+
+  [alt*="hamburger"] {
+    display: none;
+  }
+
+  .headline {
+    width: 1440px;
+    height: 721.46px;
+    background-image: url(images/Header-bg.svg);
+    background-repeat: no-repeat;
+  }
+
+  li {
+    list-style-type: none;
+    text-decoration: none;
+  }
+
+  .headline h1 {
+    width: 920px;
+    height: 122px;
+    margin: 200px 260px 0 260px;
+  }
+
+  .presentation {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    padding: 0 260px;
+    margin: 12px 0 0 0;
+  }
```

Comment:
- [ ] Let's use the correct `color` for this element as the `color` is `#344563;`.  Please check the template and add the correct `color`.

Reply:
Done